src/Conference/Infrastructure/Service/ConnectionFactory.php

```diff
@@ -35,9 +35,9 @@ public function __invoke(ContainerInterface $container): Connection
         $connection = new Connection(
             [
                 'pdo' => new PDO(
-                    $container->get('db_dsn'),
-                    $container->get('db_user'),
-                    $container->get('db_password')
+                    getenv('DB_DSN'),
```

Comment:
It SHALL still get this info from containers IMHO.

Register it under `services`